lib/wasmer-types/src/units.rs

```diff
@@ -108,9 +109,16 @@ where
     }
 }
 
-impl From<Bytes> for Pages {
-    fn from(bytes: Bytes) -> Self {
-        Self((bytes.0 / WASM_PAGE_SIZE) as u32)
+const PAGES_EXCEED_UINT32: &'static str = "Number of pages exceeds uint32 range";
```

Comment:
You don't need a constant here. The constant is used only once.

Reply:
I can inline it, no problem. Do you prefer that?